ivy/functional/frontends/torch/random_sampling.py

```diff
@@ -54,3 +54,14 @@ def multinomial(input, num_samples, replacement=False, *, generator=None, out=No
 @to_ivy_arrays_and_back
 def poisson(input, generator=None):
     return ivy.poisson(input, shape=None)
+
+
+@to_ivy_arrays_and_back
+def rand(size, *, generator=None, out=None, dtype=None, layout=None, device=None,
```

Comment:
It's fine to ignore `pin_memory`, as with `requires_grad` I'm not sure and I'm waiting to hear back from the team member who's worked on gradients the most, will let you know when I hear back

Reply:
marking as resolved for now but can reopen if requires_grad needs changing